CHANGELOG.md

```diff
@@ -1,11 +1,54 @@
 # Changelog
 
+## v1.5.0
+(Just for temporary reference, this is the last commit considered: ac4c881ad12cca669d4b7b1745d6dd5f0553a1ba)
+
+In this minor version release, the support for Python 3.9 is incorporated [[#4301]](https://github.com/aiidateam/aiida-core/pull/4301), while Python 3.5 is dropped [[#4386]](https://github.com/aiidateam/aiida-core/pull/4386).
+This version is compatible with all current Python versions that are not end-of-life:
+ * 3.6
+ * 3.7
+ * 3.8
+ * 3.9
+
+### Features
+- Process functions can now be submitted through the engine [[#4539]](https://github.com/aiidateam/aiida-core/pull/4539)
+- REST API: list endpoints at base URL [[#4412]](https://github.com/aiidateam/aiida-core/pull/4412)
+- `ProcessBuilder`: allow unsetting of inputs through attribute deletion [[#4419]](https://github.com/aiidateam/aiida-core/pull/4419)
+- `verdi migrate`: make `--in-place` work across different file systems [[#4393]](https://github.com/aiidateam/aiida-core/pull/4393)
+
+### Improvements
+- Added remaining original documentation that didn't make it into the first step of the recent major overhaul of v1.3.0
+- Archive importer: refactor the reading code [[#4510]](https://github.com/aiidateam/aiida-core/pull/4510)
+- Archive importer: refactor the migrations code [[#4532]](https://github.com/aiidateam/aiida-core/pull/4532)
+- Archive exporter: refactor progress bar [[#4504]](https://github.com/aiidateam/aiida-core/pull/4504)
+- Archive exporter: decouple the extraction of data from the writing to file [[#4448]](https://github.com/aiidateam/aiida-core/pull/4448)
```

Comment:
Since these concern mostly refactoring and so have merely improvements for developers, should they go to the `Developers` section instead?

Reply:
Yes, I was debating myself this when I was adding the new ones. I believe at the start we included the first of these here because they also had some performance improvements, but I agree it might just be better to send all of them to `Developers`.